commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/DraftBuilderGenerator.java

```diff
@@ -128,6 +130,117 @@ private void createCopyFactoryMethods(final TypeElement resourceDraftValueType,
         }
     }
 
+    private boolean isListProperty(PropertyGenModel propertyGenModel){
+        final boolean isParmetrizedList = (propertyGenModel.getType() instanceof ParameterizedTypeName) && ((ParameterizedTypeName)propertyGenModel.getType()).rawType.reflectionName().equals(List.class.getTypeName());
+        final boolean isNonParmetrizedList = (propertyGenModel.getType() instanceof ClassName) && ((ClassName)propertyGenModel.getType()).reflectionName().equals(List.class.getTypeName());
+
+        return isNonParmetrizedList || isParmetrizedList;
+    }
+
+    private List<MethodSpec> createPlusLists(final TypeElement resourceDraftValueType,final List<PropertyGenModel> propertyGenModels){
+
+        return propertyGenModels.stream()
+                .filter(this::isListProperty)
+                .map(propertyGenModel -> createPlusListMethodSpec(resourceDraftValueType,propertyGenModel))
+                .collect(Collectors.toList());
+
+    }
+
+    private MethodSpec createPlusListMethodSpec(final TypeElement resourceDraftValueType, PropertyGenModel property){
```

Comment:
The `final` modifier is missing on the last parameter.

Reply:
good catch 👍